backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -522,6 +665,12 @@ class SchedulingService implements ISchedulingService {
       retNewSchedule,
     );
 
+    // send email to admin
```

Comment:
nit:
```suggestion
```

Reply:
with a descriptive name for the function, I think we can take out this comment :)